src/core/utils.jl

```diff
@@ -162,6 +175,63 @@ function effective_cost(;
     return round(cap_cost_slope, digits=4)
 end
 
+function replacement_effective_cost(;
```

Comment:
I made this a separate function so we can just call this to get replacement costs in the reporting of results.